packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+import Text from '../text';
+
+// Todo: Create a utility type for getting a range of float number. ex: FloatRange<0, 1>
+type TRangeFloatZeroToOne = 0 | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | 1;
+
+type TProgressBarProps = {
+    label: string;
+    value: TRangeFloatZeroToOne;
```

Comment:
since you initialized `value` with 0, you can make this optional. 
```suggestion
    value?: TRangeFloatZeroToOne;
```

Reply:
@iman-fs Based on the component usage it seems that `value` is required 🤔